src/servers/model/server/api-web.ts

```diff
@@ -45,11 +45,11 @@ async function processNextJob() {
     }
 }
 
-export function createResultWriter(response: express.Response, encoding: string, entryId?: string, queryName?: string) {
+export function createResultWriter(response: express.Response, encoding: string, download: boolean, entryId?: string, queryName?: string) {
```

Comment:
If we add any more params here perhaps it should be params object instead. Would you might doing that refactoring now?

Reply:
Done in ea9e25b. Let me know if you'd like to see `ResultWriterParams` moved somewhere else.